geode-docs/configuring/running/running_the_cacheserver.html.md.erb

```diff
@@ -31,7 +31,7 @@ The Geode server is used primarily for hosting long-lived data regions and for r
 
 The `gfsh` utility uses a working directory for its configuration files and log files. These are the defaults and configuration options:
 
--   When you start a standalone server using `gfsh`, `gfsh` will automatically load the required JAR files `$GEMFIRE/lib/server-dependencies.jar` and `$JAVA_HOME/lib/tools.jar` into the CLASSPATH of the JVM process. If you start a standalone server using the ServerLauncher API, you must specify `$GEMFIRE/lib/server-dependencies.jar` inside your command to launch the process. For more information on CLASSPATH settings in Geode, see [Setting Up the CLASSPATH](../../getting_started/setup_classpath.html).
+-   When you start a standalone server using `gfsh`, `gfsh` will automatically load the required JAR file `$GEMFIRE/lib/server-dependencies.jar` into the CLASSPATH of the JVM process. If you start a standalone server using the ServerLauncher API, you must specify `$GEMFIRE/lib/server-dependencies.jar` inside your command to launch the process. For more information on CLASSPATH settings in Geode, see [Setting Up the CLASSPATH](../../getting_started/setup_classpath.html).
```

Comment:
All references to $GEMFIRE/lib/server-dependencies.jar should change to $GEODE/lib/geode-dependencies.jar